kimchi/src/prover.rs

```diff
@@ -146,10 +167,33 @@ where
                 witness[i].clone(),
                 index.cs.domain.d1,
             );
-            index
-                .srs
-                .commit_evaluations(index.cs.domain.d1, &e, None, rng)
+
+            match &blinders[i] {
+                None => index
+                    .srs
+                    .commit_evaluations(index.cs.domain.d1, &e, None, rng),
+                Some(b) => {
+                    let c = index
+                        .srs
+                        .commit_evaluations_non_hiding(index.cs.domain.d1, &e, None);
+                    (
+                        c.zip(b).unwrap().map(|(g, b)| {
```

Comment:
`c.zip(b).unwrap().map(|(g, b)|`

that's a lot of one-letter variable names :p